examples/basic-iframe/.tina/database.js

```diff
@@ -0,0 +1,17 @@
+import { createDatabase } from '@tinacms/graphql'
+import { FilesystemBridge } from '@tinacms/datalayer'
+import { MongodbLevel } from 'mongodb-level'
+import { config } from 'dotenv'
+
+config()
+
+const level = new MongodbLevel({
+  mongoUri: process.env.MONGO_URI,
+  dbName: 'tinacms',
+  collectionName: 'tinacms',
+})
+
+export default createDatabase({
+  bridge: new FilesystemBridge(process.cwd()),
```

Comment:
Having this here means the `api/gql` route has a larger bundle, including the `fs` and `path` modules, which it doesn't use. We should probably bake the `bridge` in by default, to the CLI so this isn't needed